### ISSUE TITLE
Bluetooth disconnection icon does not show on bottom sidebar

### ISSUE THREAD

**Issue by mvalchev2** [plarus/TimeStyleBBPebble#10]
When the watch is not connected to the phone and configuration is with Bottom sidebar and 3 complications (for example Battery, Weather current and Steps) - nothing changes. The expected behavior is to display a bluetooth disconnect icon in the place of the weather icon - as its is on right and left sidebars.

**Comment by plarus** [comment]
My first objective is that when you have 3 complications in bottom/top bar and Bluetooth disconnect icon must be displayed => Display Bluetooth disconnect icon as 4th complication.

I have found and corrected the problem.

Another possibility is your expected behavior: still display 3 complication and replace the middle complication by Bluetooth disconnect icon.

I think it is better to display all complication if it is possible.
Are you OK with this solution?

**Comment by mvalchev2** [comment]
Ok. One more thing - if one from 4 complications is weather widget, may be it's logical in its place to be the bluetooth disconnection widget, what do you think?

**Comment by plarus** [comment]
Yes it is already done like that: 

```
// are there any bluetooth-enabled widgets? if so, they're the second-best
  // candidates
  for(int i = 0; i < 4; i++) {
    if(globalSettings.widgets[i] == WEATHER_CURRENT || globalSettings.widgets[i] == WEATHER_FORECAST_TODAY) {
      return i;
    }
  }
```

**Comment by plarus** [comment]
Corrected in 1.1.0 release
